### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by mikemtzp**
- Create React App.
- Install React Redux and React Router.
- Add an image for the app logo.
- Create routes and view components (Rockets, Missions, My Profile). Use `<NavLink />` for the page navigation links and style active class to indicate which section/page the user is currently on (underline active navigation link).
- Create directories for all Redux state slice files (rockets and missions).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
-# Space-Travelers-Hub
\ No newline at end of file
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
```
Comment (Zeraltz) [inline README.md:5]: Please remove the README CRA boilerplate, add your own

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <!--
+      manifest.json provides metadata used when your web app is installed on a
+      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
+    -->
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!--
```
Comment (Zeraltz) [inline public/index.html:18]: Remove boilerplate comments on the HTML document

**Review by Zeraltz — CHANGES_REQUESTED** [review]
Hello, the project's looking good so far, there are a couple of changes that I believe are necessary for cleaner code but the functionality's looking good to me.

Please let me know if you have any questions about it.

File: README.md (hunk 1 above)
Comment (mikemtzp) [reply README.md]: Hello! I have updated the READme.md file. 👍🏼

File: public/index.html (hunk 2 above)
Comment (mikemtzp) [reply public/index.html]: Boilerplate deleted successfully! Thanks for noting it.

**Review by Zeraltz — APPROVED** [review]
LGTM.

Time to merge

**Pull request merged**